### PR TITLE
WEIS

### PR DESCRIPTION
**Pull request by nikhar-abbas**
Updates for WEIS. 

A few updates here:
1) All of the aeroelasticSE tools are now supported under the [weis](https://github.com/wisdem/weis) project now, so changes were made to the necessary files here to accommodate them. 
2) OpenFAST changed how the binaries are written, so the openfast output reader was updated. 
3) Large improvements to the testing suite 
- "lite" and "heavy" tests to run subsets of DLCs 
- "discon comp" and "binary comp" functions to compare DISCON.IN files or compiled binaries
- `run_testing.py` to run a standard set of tests with some high-level post processing. NOTE: The post processing here can and should be expanded a bit more.

### CODE REVIEW

File: ROSCO_testing/ROSCO_testing.py [hunk 1]
```diff
@@ -296,6 +304,8 @@ def ROSCO_Test_heavy(self, more_case_inputs={}, U=[]):
         iec.dlc_inputs['Yaw'] = [[], []]
         iec.transient_dir_change = '-'  # '+','-','both': sign for transient events in EDC, EWS
         iec.transient_shear_orientation = 'v'  # 'v','h','both': vertical or horizontal shear for EWS
+        iec.uniqueSeeds = True
```
Comment (nikhar-abbas) [inline ROSCO_testing/ROSCO_testing.py:307]: This seems like it needs the unique seeds related pull request to go through in WEIS - yes?

**Comment by nikhar-abbas** [comment]
This is pretty close to ready. A short to do list, that could be added to:

- [ ] update READMEs for WEIS
- [ ] check tuning scripts and distributed yaml files for "good" controller tuning.

File: ROSCO_testing/ROSCO_testing.py (hunk 1 above)
Comment (dzalkind) [reply ROSCO_testing/ROSCO_testing.py:308]: Having this here without the related pull request in WEIS will not break anything, but it won't generate unique wind seeds without that PR.

**Comment by dzalkind** [comment]
To follow up on the two check boxes:
- update READMEs for WEIS -> let's skip this since we are removing the dependency on WEIS soon
- check tuning scripts and distributed yaml files for "good" controller tuning. -> this looks fine for the NREL-5MW and IEA-15MW

**Pull request merged**